modules/administration/pages/content-lifecycle.adoc

```diff
@@ -107,7 +107,7 @@ Each filter has a ``rule`` parameter that can be set to either ``Allow`` or ``De
 The filters are processed like this:
 
 * If a package or patch satisfies a ``Deny`` filter, it is excluded from the result.
-* If a package or patch satisfies an ``Allow`` filter, it is included in the result (even if it was excluded by a ``Deny`` filter).
+* If a package or patch satisfies an ``Allow`` filter, it is included in the result (even if it was excluded by a ``Deny`` filter). ``Allow`` filters for packages only operate on package filters and ``Allow`` filters for patches only operate on patch filters, i.e. a package filter can't be used to add packages from patches that where filtered out back in and vise versa.
```

Comment:
I'm not sure whether I got it right and my suggestion is correct.  It is just a proposal.
```suggestion
* If a package or patch satisfies an ``Allow`` filter, it is included in the result (even if it was excluded by a ``Deny`` filter).
  ``Allow`` filters for packages only operate on package filters and ``Allow`` filters for patches only operate on patch filters.
  This means with a package filter you cannot add packages back in or vice versa from patches that where filtered out.
```

Reply:
@ke - as agreed before, we can accept and merge the contributions that come from outside documentation squad, then format them in a separate PR. I'll complete this task.

Reply:
Ok, but it not just formatting, I also proposed to change sentence structure and the wording.  To make this visible, I'll now try to revert the formatting changes.